setup.py

```diff
@@ -1,6 +1,7 @@
 import os
 import warnings
 from packaging import version
+import versioneer
 import pip
 from setuptools import setup
 from setuptools import Command
```

Comment:
Is this an old version of the `setup.py`? Where are the `try` blocks etc.?

Reply:
https://github.com/precice/python-bindings/pull/63 is not merged yet. You mean these `try` blocks, I assume?

Reply:
Ah yes, that is what I meant. I guess I am just too used to the "new" `setup.py.`